hitbox-tokio/src/runtime.rs

```diff
@@ -80,4 +80,12 @@ where
         };
         hitbox_backend::EvictionPolicy::Ttl(ttl_settings)
     }
+
+    fn upstream_name(&self) -> Cow<'static, str> {
```

Comment:
Why not in default implementation of `RuntimeAdapter` trait?

Reply:
I can't find out upstream name before trait implementation